Basic-Car-Maintenance/Documentation.docc/getting-started.tutorial

```diff
@@ -88,6 +88,53 @@
             }
         }
     }
+
+     @Section(title: "Setting Up Firebase Local Emulator") {
```

Comment:
while this now renders better, many new warnings are created because using underscores doesn't fix this issue. Refer to the documentation to properly format the document


![Screenshot 2024-10-01 at 9 31 10 PM](https://github.com/user-attachments/assets/b371d1fd-6e66-4455-8f2a-fc805b9b9f0d)